client/bulkwriter/bulk_import.go

```diff
@@ -0,0 +1,313 @@
+// Licensed to the LF AI & Data foundation under one
+// or more contributor license agreements. See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership. The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License. You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package bulkwriter
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"fmt"
+	"io"
+	"net/http"
+)
+
+type ResponseBase struct {
+	Status  int    `json:"status"`
+	Message string `json:"message"`
+}
+
+func (b ResponseBase) CheckStatus() error {
+	if b.Status != 0 {
+		return fmt.Errorf("bulk import return error, status: %d, message: %s", b.Status, b.Message)
+	}
+	return nil
+}
+
+type BulkImportOption struct {
+	// milvus params
+	URI            string     `json:"-"`
+	CollectionName string     `json:"collectionName"`
+	Files          [][]string `json:"files"`
```

Comment:
`Files` should be optional, as in the cloud, the import files are specified using `ObjectURL` instead.